sdk/eventhub/Microsoft.Azure.Management.EventHub/src/Generated/SdkInfo_EventHubManagementClient.cs

```diff
@@ -19,26 +19,29 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("EventHub", "Clusters", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "Configuration", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "ConsumerGroups", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "DisasterRecoveryConfigs", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "EventHubs", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "Namespaces", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "Operations", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "PrivateEndpointConnections", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "PrivateLinkResources", "2021-11-01"),
-                new Tuple<string, string, string>("EventHub", "SchemaRegistry", "2021-11-01"),
+                new Tuple<string, string, string>("EventHub", "ApplicationGroup", "2022-01-01-preview"),
```

Comment:
It seems that we are going to release a preview sdk, please also update following files:

- Version and ReleaseNote in [csproj](https://github.com/damodaravadhani/azure-sdk-for-net/blob/eventHubs2022SDK/sdk/eventhub/Microsoft.Azure.Management.EventHub/src/Microsoft.Azure.Management.EventHub.csproj#L12-L28)
- AssemblyVersion and AssemblyFileVersion in [AssemblyInfo.cs](https://github.com/damodaravadhani/azure-sdk-for-net/blob/eventHubs2022SDK/sdk/eventhub/Microsoft.Azure.Management.EventHub/src/Properties/AssemblyInfo.cs#L10-L11)
- [AzSdk.RP.props](https://github.com/damodaravadhani/azure-sdk-for-net/blob/eventHubs2022SDK/sdk/eventhub/Microsoft.Azure.Management.EventHub/AzSdk.RP.props)

Reply:
Hi @damodaravadhani, please address this comment. thanks.

Reply:
Hey @Yao725 , we found a bug in our swagger documentation and I have raised a PR to fix it. Once it's merged I will push the SDK change as well.